playground/src/components/custom-views/demo-custom-view.jsx

```diff
@@ -78,7 +88,7 @@ function DemoCustomView() {
   }
 
   return (
-    <CustomViewShell customViewId={CUSTOM_VIEW_ID} messages={{}}>
+    <CustomViewShell disableDevHost applicationMessages={{}}>
```

Comment:
Since we are rendering the `<CustomViewDevHost>` from within the `<CustomViewShell>`, we need to explicitly opt out of it even in development, as the setup we have in playground does not need that component.